commons-extension-webapp/src/main/webapp/eXoPlugins/tagSuggester/plugin.js

```diff
@@ -29,7 +29,24 @@ require(['SHARED/jquery', 'SHARED/tagSuggester'],function($) {
   }
 
   CKEDITOR.plugins.add( 'tagSuggester', {
+    // Register the icons. They must match command names.
+    icons: 'tagSuggester',
+    lang : ['en','fr'],
     init : function( editor ) {
+      editor.addCommand( 'tagSuggester', {
+      // Define the function that will be fired when the command is executed.
+        exec: function( editor ) {
+          document.dispatchEvent(new CustomEvent('test-custom-plugins'));
```

Comment:
`test-custom-plugins` should be renamed IMO

Reply:
Done !